packages/nexus-sdk/src/Archive/index.ts

```diff
@@ -0,0 +1,50 @@
+import { Fetchers } from '../types';
+import { NexusContext } from '../nexusSdk';
+import { ArchivePayload, GetArchiveOptions } from './types';
+import { buildQueryParams } from '../utils';
+
+const Archive = (
+  { httpGet, httpPut, httpPost, poll }: Fetchers,
+  context: NexusContext,
+) => {
+  return {
+    get: (
+      orgLabel: string,
+      projectLabel: string,
+      archiveId: string,
+      options?: GetArchiveOptions,
+    ): Promise<string> => {
+      const opts = { as: 'text', ...options };
+      const acceptHeader =
+        opts.as === 'json' ? 'application/ld+json' : 'application/x-tar';
+      return httpGet({
+        headers: { Accept: acceptHeader },
+        path: `${
+          context.uri
+        }/archives/${orgLabel}/${projectLabel}/${archiveId}${buildQueryParams(
+          opts,
+        )}`,
+      });
+    },
+    create: (
+      orgLabel: string,
+      projectLabel: string,
+      payload: ArchivePayload,
+    ): Promise<string> => {
```

Comment:
The Promise type is more

```suggestion
    ): Promise<Archive | Blob | string> => {
```